task6/Working with posts.js

```diff
@@ -0,0 +1,251 @@
+(function(){
+class post {
+	constructor(id, avatar, author, createdAt, description, hashTags=[], photoLink="") {
+		this.id = id;
+		this.avatar = avatar;
```

Comment:
I'm not sure, that post should contain author's avatar

Reply:
In my project post contains the author's avatar

Reply:
I'm talking not about "in TR there were no avatars", but about project's architecture. It looks wrong, that post contains avatar. It already contains author's identifier, avatar should be in some userInfo structure.